crates/re_data_store/src/store_db.rs

```diff
@@ -347,9 +347,13 @@ impl StoreDb {
         re_tracing::profile_function!();
         assert!((0.0..=1.0).contains(&fraction_to_purge));
 
-        let (drop_row_ids, stats_diff) = self.entity_db.data_store.gc(
-            re_arrow_store::GarbageCollectionTarget::DropAtLeastFraction(fraction_to_purge as _),
-        );
+        let (drop_row_ids, stats_diff) = self.entity_db.data_store.gc(GarbageCollectionOptions {
+            target: re_arrow_store::GarbageCollectionTarget::DropAtLeastFraction(
+                fraction_to_purge as _,
+            ),
+            gc_timeless: false,
+            protect_latest: 0,
```

Comment:
What do you think about actually turning these on? Save for a separate PR?

```suggestion
            gc_timeless: true,
            protect_latest: 1,
```

Reply:
Yeah, I wanted to be careful about introducing perf or behavioral changes in a patch-release, but def want to do that in a followup PR.